### PR TITLE
Editor can navigate through the admin website 

### PR DESCRIPTION
**Pull request by Fraseer**
PT: https://www.pivotaltracker.com/story/show/179900628

## User Story
```
As an editor,
In order to get to different parts of the website
I want to set up the basic navigation functionality  
```

## Tasks
- Set up routes with react-router-dom
- Create Header 
- Add a Create Articles and Review page

## Screenshot

image

image

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (lukeperrin10) [inline cypress/fixtures/example.json]: do you need this?

File: src/App.jsx [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 const App = () => {
   return (
-    <div>
-      <h1>Hello Admin World</h1>
-    </div>
+    <BrowserRouter>
+      <Header />
+      <Switch>
+        <Route exact path="/" component={HomePage}></Route>
+        <Route exact path="create-article" component={CreateArticle}></Route>#
```
Comment (lukeperrin10) [inline src/App.jsx]: typo I am guessing with the extra # at the end

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { Link } from "react-router-dom";
+
+const Header = () => {
+  return (
+    <Link
+      to={{
+        pathname: "/",
+      }}
+    >
+      <header className="siteName" data-cy="header">
```
Comment (maxarvid) [inline src/components/Header.jsx]: Here is where I meant to for the less code comment to go: Maybe consider doing the `<header>` as an `as={Link}` ?

File: src/App.jsx (hunk 2 above)
Comment (Fraseer) [reply src/App.jsx]: It is, removed

File: cypress/fixtures/example.json (hunk 1 above)
Comment (Fraseer) [reply cypress/fixtures/example.json]: We do not, shall be removed

File: src/components/Header.jsx (hunk 3 above)
Comment (Fraseer) [reply src/components/Header.jsx]: That only works for semantics tags I believe

**Comment by lukeperrin10** [comment]
Even though it is an Admin site, it should still have a user story.

File: src/components/Header.jsx (hunk 3 above)
Comment (maxarvid) [reply src/components/Header.jsx]: Duly noted, I'll resolve this convo

**Comment by Fraseer** [comment]
Ah okay I thought as it was a chore it didnt require a user story, so took it out

File: src/styles/index.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.siteName {
```
Comment (tochman) [inline src/styles/index.css]:
```
site-name
```

**Pull request merged**